src/mavedb/lib/logging.py

```diff
@@ -78,11 +78,17 @@ async def logging_route_handler(request: Request) -> Response:
             end = time.time_ns()
 
             # Add logging task to list of background tasks to be completed after the response is sent.
-            tasks = response.background
+            old_background = response.background
             task = BackgroundTask(log_info, request, response, start, end)
-            if tasks:
-                tasks.add_task(task)
-                response.background = tasks
+            if old_background:
+                if isinstance(old_background, BackgroundTasks):
+                    old_background.add_task(task)
+                    response.background = old_background
+                else:
```

Comment:
What if old_background is None? Should this become a third case, where we just say `response.background = task`?

Reply:
if old_background is None, the if statement at L83 should be false, then see the else at L92

Reply:
Yep, thanks. I didn't read everything...